css/style.css

```diff
@@ -46,14 +46,47 @@ body {
 }
 
 .menu__item {
-  padding: 0 13px;
-  font-family: 'Roboto', sans-serif;
-  font-weight: 500;
-  font-size: 18px;
+  position: relative;
+  margin-right: 26px;
+  font: 500 18px/21px 'roboto-medium', sans-serif;
+  color: #000;
+  text-decoration: none;
+  transition: opacity .5s;
+}
+
+.menu__item_active::after {
+  position: absolute;
+  display: block;
+  content: '';
+  left: 0;
+  width: 100%;
+  height: 3px;
+  background: #0056EC;
```

Comment:
Codacy found an issue: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/html_abrmhah/pullRequest?prid=6313902)